helm/reana/values.yaml

```diff
@@ -71,6 +71,7 @@ components:
       REANA_SCHEDULER_REQUEUE_COUNT: 200
       REANA_USER_EMAIL_CONFIRMATION: true
       REANA_WORKFLOW_SCHEDULING_POLICY: "fifo"
+      REANA_WORKFLOW_SCHEDULING_READINESS_CHECK: 9
```

Comment:
What about naming the variable `REANA_WORKFLOW_SCHEDULING_READINESS_CHECK_LEVEL` to make an analogy to "log levels" etc that people might be accustomed to already?  Basically, "check level 0" and "check lever 9" is easier to picture, whilst "check 0" and "check 9" may be somewhat misleading...